database/changesets/20210414_Sarscov2IlluminaFullTable.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="chengche@broadinstitute.org"
+               dbms="postgresql"
+               id="20210414"
+               logicalFilePath="20210414_Sarscov2IlluminaFullTable.xml"
+               runInTransaction="false">
+        <comment>
+            the Sarscov2IlluminaFull table schema
+        </comment>
+        <sql>
+            CREATE TYPE Sarscov2IlluminaFull AS (
+            id                          bigint,      -- just an unique id
+            dataset_id                  text,        -- Terra DataRepo dataset id
+            snapshot_job_id             text,        -- Terra DataRepo snapshot creation job id
+            snapshot_id                 text,        -- Terra DataRepo snapshot id
+            datarepo_row_id             text,        -- Unique id to identify a row in snapshot
+            submission_id               text,        -- FireCloud submissionId
+            workflow_id                 text,        -- FireCloud workflow id
+            status                      text,        -- FireCloud workflow status
```

Comment:
Is this value a candidate for an enum definition (not sure how widely we use these)?

Reply:
That's a good point! If FireCloud's workflows are similar to Cromwell workflows, then this will be a finite set of strings (`Failed`, `Succeeded`, `Running`, `On Hold`, etc.). Historically we have just used plain text for this field, so I inherited that behavior. Maybe we could turn this into an Enum now?

Reply:
If not too much trouble to change, that would be my vote.  When I see a finite set of strings I like enums in part for the built-in explanation they provide on expected / supported values.

If you'd rather merge now in the name of expediency and revisit later (if not too difficult to change an existing schema) then fine by me as well.

Reply:
On the other hand, WFL doesn't control those values.
Enumerating them means a Firecloud change requires a WFL schema change.
On the third hand, WFL probably uses those status values in code too.
On the fourth hand, WFL code changes are somewhat easier than database schema changes.

Reply:
Given the count of hands here, I'll approve the PR and we can elect to discuss further later.

Reply:
Sounds good, thank you both!!